assets/js/app.js

```diff
@@ -1,18 +1,54 @@
 var $ = require('jquery');
 var _ = require('underscore');
+var socketio = require('socket.io-client');
+var tabs = require('./vendor/jquery.tabs');
+var sailsIOClient = require('sails.io.js');
 
-var view = require('./views/dashboard');
+var view = require('./views/centre');
 
 $(function() {
-  var options = {};
+  var io;
+  var socket0;
+  var container = $('#centres');
   var params = document.location.search.replace("?","").split("&");
-  var endpointParam = _.find(params, function(item) {
-    return item.match('endpoint');
+  var simulatorParam = _.find(params, function(item) {
+    return item.match('simulator');
   });
 
-  if (endpointParam) {
-    options.url = endpointParam.split("=")[1];
+  if (simulatorParam) {
+    var simulator = require('./socketSimulator');
+    socketio = simulator.client;
   }
 
-  new view(options);
+  io = sailsIOClient(socketio);
+
+  io.sails.autoConnect = false;
+  if (!simulatorParam) io.sails.url = 'http://localhost:8080';
+
+  socket0 = io.sails.connect();
+
+  socket0.get('/centre', function serverResponded(payload) {
```

Comment:
this won't handle centres being added or removed while the socket is up, can this not databind instead rather than init containers for each centre on startup?